yjit.c

```diff
@@ -509,6 +509,15 @@ rb_get_cme_def_body_attr_id(const rb_callable_method_entry_t *cme)
     return cme->def->body.attr.id;
 }
 
+
+ID rb_check_id2(VALUE *namep);
```

Comment:
This declaration needs adjustment now. You can also remove `rb_get_symbol_id()` which is giving compile warnings.